runelite-client/src/main/java/net/runelite/client/plugins/microbot/thieving/ThievingScript.java

```diff
@@ -73,6 +62,23 @@ public boolean run(ThievingConfig config) {
         return true;
     }
 
+    private void handeFood() {
+        List<Rs2Item> foods = Rs2Inventory.getInventoryFood();
+
+        if (foods.isEmpty()) {
+            openCoinPouches(1);
+            bank();
+            return;
+        }
+        if (Rs2Inventory.isFull()) {
```

Comment:
I've seen that this should not be part of this method I'll get it out otherwise it will cause a bug not to drop items. Looking into it right now.